src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Protocol/DelegatedTypes.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace Microsoft.CodeAnalysis.Razor.Protocol;
 
+using Location = Roslyn.LanguageServer.Protocol.Location;
```

Comment:
Similar to `LspRange`, consider moving this above the namespace declaration and renaming `Location` to `LspLocation` to avoid conflicts.